### ISSUE TITLE
Error on ESM feature

### ISSUE THREAD

**Issue by Murzbul** [a7ul/esbuild-node-tsc#47]
Hi how are things?

First of all thank you for this wonderful package.

It seems that the last change you sent is breaking a part of the package since by adding the types module, you are using `require` in the config.js file.

![image](https://user-images.githubusercontent.com/7916055/182953774-1234d940-538f-49b8-932d-b33761b4cfcc.png)

![image](https://user-images.githubusercontent.com/7916055/182953917-0e306aec-1e5a-46f3-a999-d70987c6ec41.png)

When I went back to version 1.8.6 it compiled without problems.

**Comment by a7ul** [comment]
Ah crap! 
Let me disable that release. Thanks for raising it!

**Comment by Murzbul** [comment]
> Ah crap! Let me disable that release. Thanks for raising it!

No problem!

I just realized it because I was about to release the new version of a CLI that uses this package.

https://github.com/DigiChanges/nexp-cli

**Comment by joaquimnetocel** [comment]
Only version 1.8.6 working here too. Hope you can fix it soon :)

**Comment by a7ul** [comment]
This is now fixed in v2.0
https://github.com/a7ul/esbuild-node-tsc/tree/v2.0.0#v20-migration-and-breaking-changes-

**Comment by Murzbul** [comment]
> Only version 1.8.6 working here too. Hope you can fix it soon :)

Thanks for the comment.

It is exactly what I did that day that I posted the error, I needed to launch the CLI at that moment.

At some point in the week I hope to migrate to the new stable version.

**Comment by joaquimnetocel** [comment]
The problem seems to continue in version 2.0.1.

**Comment by a7ul** [comment]
Hmm but there is no require in v2.0.1 🤔 
Can you post the stacktrace or log?

**Comment by joaquimnetocel** [comment]
Here is a reproduction of the problem: https://github.com/joaquimnetocel/problemEsbuildNodeTsc

If you use version 1.8.6 on the project above, everything works fine. With version 2.0.1, it gives an error.

**Comment by a7ul** [comment]
Hey @joaquimnetocel 
I tried the repo but I got few unrelated errors:
<img width="487" alt="image" src="https://user-images.githubusercontent.com/4029423/184885780-c5c26966-259d-4b9f-b22c-1a4d97fd3bd1.png">
```
./folderDist/index.js
```
This file doesnt exist so it complains.
But instead if I do 
```
npx esbuild-node-tsc --config=esbuild.development.config.cjs
```
Things do work with a warning telling to upgrade the config
```
❯ npx esbuild-node-tsc --config=esbuild.development.config.cjs
Your etsc config file is using the old v1.0 format. Please update to the new v2.0 format. Check https://github.com/a7ul/esbuild-node-tsc for details.
Built in: 43.544ms
```

After updating the config to latest format:
```js
module.exports = {
	esbuild: {
		minify: false,
		target: "es2021",
		format: "esm",
		platform: "node",
	},
};
```
Things work without any warning
<img width="642" alt="image" src="https://user-images.githubusercontent.com/4029423/184886553-68bc4662-dd7f-496c-b12c-90f65d90fb56.png">

**Comment by joaquimnetocel** [comment]
Here is what happens when I try to do `npx esbuild-node-tsc --config=esbuild.development.config.cjs` on my computer. I am using windows.

![esbuild](https://user-images.githubusercontent.com/42927465/184890589-4312a392-69d3-40b4-aaf2-b465a38c0da4.png)

**Comment by joaquimnetocel** [comment]
It seems to be something related to the operational system (Windows) or path resolvers, because (as you can see in the previous message) on your computer it is working and on mine it is not. As I said, version 1.8.6 runs fine in my computer while recent versions give the error I printed.

**Comment by a7ul** [comment]
Interesting, which node version are you using ?

**Comment by joaquimnetocel** [comment]
Node version 16.16.0

**Comment by a7ul** [comment]
This should be fixed now in 2.0.2 
Could you verify it in windows too?

**Comment by joaquimnetocel** [comment]
Great job! Now it is working on Windows too!
